presto-product-tests/bin/run_on_docker.sh

```diff
@@ -89,6 +94,10 @@ if [[ ! -f "$DOCKER_CONF_LOCATION/$ENVIRONMENT/compose.sh" ]]; then
   usage
 fi
 
+if [[ "$ENVIRONMENT" == "fileauth" ]]; then
+    CLI_ARGUMENTS="--server https://presto-master:8443 --user userCrypt --password --truststore-path /etc/ldap/cacerts.jks --truststore-password testldap"
```

Comment:
define this in `presto-product-tests/conf/docker/fileauth/docker-compose.yml`

Reply:
it's already there https://github.com/prestosql/presto/pull/1913/commits/e61633ddd5a3fd02fc8fb84bc83741ca8c5e7348#diff-26231eb27080877fabe76bdf699c4e39R14

so should be enough to remove it from here